diff --git a/src/rn/ExpedirProcedimentoRN.php b/src/rn/ExpedirProcedimentoRN.php
index 5f889b7c..d067b283 100755
--- a/src/rn/ExpedirProcedimentoRN.php
+++ b/src/rn/ExpedirProcedimentoRN.php
@@ -1785,6 +1785,7 @@ class ExpedirProcedimentoRN extends InfraRN {
       $objProcedimentoDTO->retDblIdProcedimento();
       $objProcedimentoDTO->retNumIdHipoteseLegalProtocolo();
       $objProcedimentoDTO->retStrProtocoloProcedimentoFormatadoPesquisa();
+      $objProcedimentoDTO->retStrIdProtocoloFederacaoProtocolo();
 
       return $this->objProcedimentoRN->consultarRN0201($objProcedimentoDTO);
     }
@@ -2510,7 +2511,7 @@ class ExpedirProcedimentoRN extends InfraRN {
     private function validarProcedimentoCompartilhadoSeiFederacao(InfraException $objInfraException, $objProcedimentoDTO, $strAtributoValidacao) {
       $bolProcedimentoCompartilhado = false;
       $objProtocoloFederacaoDTO = new ProtocoloFederacaoDTO();
-      $objProtocoloFederacaoDTO->setStrProtocoloFormatadoPesquisa($objProcedimentoDTO->getStrProtocoloProcedimentoFormatadoPesquisa());
+      $objProtocoloFederacaoDTO->setStrIdProtocoloFederacao($objProcedimentoDTO->getStrIdProtocoloFederacaoProtocolo());
       $objProtocoloFederacaoDTO->retStrProtocoloFormatado();
 
       $objProtocoloFederacaoRN = new ProtocoloFederacaoRN();
